test/web/helpers.js

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {TokenService} from 'bedrock-web-authn-token';
+import {AccountService} from 'bedrock-web-account';
+import {authenticator} from 'otplib';
+import {MemoryEngine} from 'bedrock-web-store';
+
+export const store = new MemoryEngine();
+const tokenService = new TokenService();
+const accountService = new AccountService();
+
+export async function login({email, password, totp}) {
+  const authResults = {};
+  const challenge = authenticator.generate(totp.secret);
+  authResults.totp = await tokenService.authenticate(
+    {type: 'totp', email, challenge});
+  authResults.password = await tokenService.authenticate(
+    {email, type: 'password', challenge: password});
+  authResults.login = await tokenService.login();
+  return authResults;
+}
+
+export async function createAccount({
+  email,
+  password,
+  short_name = 'session-test'
+}) {
+  // check to make sure the account does not already exist.
+  const exists = await accountService.exists({email});
+  if(exists) {
+    // if the account exists just return
+    return;
+  }
+  const account = await accountService.create({email});
+  await tokenService.setAuthenticationRequirements({
+    account: account.id,
+    requiredAuthenticationMethods: [
+      'totp-test-challenge',
+      'password-test'
+    ]
+  });
+  const {result: totp} = await tokenService.create({
+    account: account.id,
+    type: 'totp',
+    authenticationMethod: 'totp-test-challenge',
+    serviceId: short_name
+  });
+  await tokenService.create({
+    account: account.id,
+    type: 'password',
+    password,
+    authenticationMethod: 'password-test',
+    serviceId: short_name
+  });
+  return {account, email, password, totp};
```

Comment:
returning values that have just been passed into this function: `email`, `password` is an anti-pattern.  Please only return new data that is generated by this function.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-session/pull/9/commits/94f6ce22182e0830e69a81e9f20db591ea4337e4